lmctl/client/api/object_groups.py

```diff
@@ -0,0 +1,18 @@
+from typing import List, Dict
+from .tnco_api_base import TNCOAPI
+from lmctl.client.client_request import TNCOClientRequest
+
+class ObjectgroupsAPI(TNCOAPI):
+    endpoint = 'api/v1/object-groups'
+
+    def get(self, id: str, shallow: bool = None) -> Dict:
```

Comment:
Do we need the shallow argument here? Does the API support that query param?

Reply:
Done.